Created a tests for class bowling

jorge27 opened the pull request:
The class bowling most have two methods, capture and calculate.
The method capture receive two values and validate if this values are
a numbers valids for append a punctuation
The method calculate, will calcule the punctuation of the shot

jorge27 committed:
Created a tests for class bowling

The class bowling most have two methods, capture and calculate.
The method capture receive two values and validate if this values are
a numbers valids for append a punctuation
The method calculate, will calcule the punctuation of the shot

EsmeSantoyo commented on spec/bowling_spec.rb:
```diff
@@ -0,0 +1,123 @@
+			end
+		end
+	end
+end
```
You must add a blank line here.
```suggestion
end

```

EsmeSantoyo reviewed (changes requested):
It’s a convention using single-quoted strings when you don't need string interpolation or special symbols, check the indentation, and remember that test files are also ruby. c:

jorge27 committed:
Correct identation, and changes some spec values

Correct identation, now are 2 spaces, change some spec values, now
instead of 0, spec nil
